src/components/question/Question.tsx

```diff
@@ -5,35 +5,55 @@ import {getCurrentDate} from '$utils/date'
 import {useLetterStore} from '$contexts/StoreContext'
 import {observer} from 'mobx-react-lite'
 
-const INITIAL_ID = 0
+const INITIAL_ID = 1
 
 const NewLetterQuestion = ({questions}: {questions: Question[]}) => {
     const {questionId, setQuestionId} = useLetterStore()
 
-    const onClickNext = () => {
-        if (questionId === Object.keys(questions).length - 1) {
-            setQuestionId(INITIAL_ID)
-        } else {
-            setQuestionId(questionId + 1)
-        }
+    const clickNextQuestion = () => {
+        if (questionId !== null) {
+            if (questionId === Object.keys(questions).length - 1) {
```

Comment:
이게 자유롭게 쓰기 맞을까요? :) 맞다면 변수로 표현해주시는것도 좋을거 같습니다 ㅎㅎ
```suggestion
const isFreeWrite = questionId === Object.keys(questions).length - 1
if (isFreeWrite) {
```

Reply:
아! 얘는 자유롭게 쓰기는 아니고, 질문 리스트의 맨 끝으로 가면 다시 앞으로 돌아오는 거예요!! 얘도 변수를 줄까요??

Reply:
아하! 네네 :) 조건문만 있으면 나중에 햇갈릴거 같아서 주시면 좋을거 같습니다 ㅎㅎ